lib/puma/dsl.rb

```diff
@@ -105,7 +105,7 @@ def activate_control_app(url="auto", opts={})
       end
 
       if opts[:no_token]
-        auth_token = :none
+        auth_token = 'none'
```

Comment:
Please add a comment here about why it's important we use a string rather than a symbol. I could see someone trying to "optimize" this later :D

Reply:
Shouldn't there be tests that fail if changed?

Reply:
@dentarg the test already checks if the value is `'none'` https://github.com/puma/puma/pull/1803/files#diff-5199cb8753ebe54bb452866fea92a0d7R44

If you think there're more cases we should cover I'm willing to know.

Reply:
I see what you mean 'dentarg' - yes I know the tests will fail, but still I'd like a record of *why* this is important for the future.